layout/_partials/comments.swig

```diff
@@ -2,8 +2,60 @@
   {%- if theme.injects.comment.length == 1 %}
     {%- set inject_item = theme.injects.comment[0] %}
     {{ partial(inject_item.layout, inject_item.locals, inject_item.options) }}
+  {%- elif theme.injects.comment.length > 1 %}
+    {%- if theme.comments.style == 'buttons' %}
+      <div class='comment-button-group'>
+        {%- for inject_item in theme.injects.comment %}
+          <a class='btn comment-button {{ inject_item.locals.class }}'>{{ inject_item.locals.button }}</a>
+        {%- endfor %}
+      </div>
+      {%- for inject_item in theme.injects.comment %}
+        <div class='comment-position {{ inject_item.locals.class }}'>
+          {{ partial(inject_item.layout, inject_item.locals, inject_item.options) }}
+        </div>
+      {%- endfor %}
+      <script{{ pjax }}>
+        (function() {
+          let commentButton = document.querySelectorAll('.comment-button');
+            commentButton.forEach(item => {
+            let commentClass = item.classList[2];
+            item.addEventListener("click", () => {
+              commentButton.forEach(rmActive => rmActive.classList.remove('active'));
+              item.classList.add('active');
+              document.querySelectorAll('.comment-position').forEach(rmActive => rmActive.classList.remove('active'));
+              document.querySelector(`.comment-position.${commentClass}`).classList.add('active');
+              {%- if theme.comments.storage %}
+                localStorage.setItem('comments_active', commentClass);
+              {%- endif %}
+            })
+          })
+          let activeClass = '{{ theme.comments.activeClass }}';
```

Comment:
`theme.comments.activeClass` is not defined

Reply:
![image](https://user-images.githubusercontent.com/15902347/62592452-c73ab800-b905-11e9-84f6-ed617e581c58.png)
In here, set `activeClass`. Make sure class name is `facebook-comments-plugin`, and config key name is `facebook_comments_plugin`